### PR TITLE
Update xenonhd.dependencies

### PR DESCRIPTION
**Pull request by NishantNalawade**
Add dependencies

### CODE REVIEW

**Comment by danparidae** [comment]
https://github.com/TeamHorizon/platform_manifest#xenonhddependencies-sample

**Review by NishantNalawade — COMMENTED** [review]
Update the dependecies

**Review by NishantNalawade — COMMENTED** [review]
Device Tree is taken from lineagerepo as it is being updated 
Please fork vendor tree from here :- https://github.com/BlackFoxy757/android_device_asus_Z00ED

File: xenonhd.dependencies [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+    "target_path": "device/asus/msm8916-common",
+    "branch": "n"
+  }
+  {
+    "repository": "LineageOS/android_kernel_asus_msm8916",
+    "target_path": "kernel/asus/msm8916",
+    "branch": "n"
+  }
+  {
+    "repository": "BlackFoxy757/proprietary_vendor_asus",
```
Comment (danparidae) [inline xenonhd.dependencies:13]: this one should be forked as well. 
btw at the moment it's completely wrong as BlackFoxy757/proprietary_vendor_asus repo doesn't exist

File: xenonhd.dependencies [hunk 2]
```diff
@@ -1,2 +1,16 @@
 [
+  {
+    "repository": "TeamHorizon/android_device_asus_qcom_common",
+    "target_path": "device/asus/msm8916-common",
+    "branch": "n"
+  }
+  {
+    "repository": "LineageOS/android_kernel_asus_msm8916",
```
Comment (danparidae) [inline xenonhd.dependencies:8]: it should be TeamHorizon, not LineageOS as we use our forks

**Pull request merged**